File: src/csharp/src/Taikor.Opensdk/TaikorToken.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Taikor.Opensdk
{
    /// <summary>
    /// Taikor 用于返回的Token类
    /// </summary>
    public class TaikorToken
    {
        /// <summary>
        /// Access Token, 
        /// </summary>
        public string AccessToken { get; set; }
        /// <summary>
        /// 错误信息
        /// </summary>
        public string Error { get; set; }
        /// <summary>
        /// 过期时间
        /// </summary>
        public long ExpiresIn { get; set; }
        /// <summary>
        /// Http错误信息
        /// </summary>
        public string HttpErrorReason { get; set; }
        /// <summary>
        /// Http错误状态码
        /// </summary>
        public int HttpErrorStatusCode { get; set; }
        /// <summary>
        /// 是否错误
        /// </summary>
        public bool IsError { get; set; }
        /// <summary>
        /// 是否Http错误
        /// </summary>
        public bool IsHttpError { get; set; }
        /// <summary>
        /// Refresh Token
        /// </summary>
        public string RefreshToken { get; set; }
        /// <summary>
        /// Token Type
        /// </summary>
        public string TokenType { get; set; }
    }
}

File: src/java/src/main/java/com/taikor/opensdk/TaikorToken.java
package com.taikor.opensdk;

/**
 * Created by Carey on 2016/7/13.
 *
*/
class TaikorToken {
    /**
    * Access Token
    */
    public String AccessToken;
    /**
    * 错误信息
    */
    public String Error;
    /**
    * 过期时间
    */
    public long ExpiresIn;
    /**
    * Http错误信息
    */
    public String HttpErrorReason;
    /**
    * Http错误状态码
    */
    public int HttpErrorStatusCode;
    /**
    * 是否错误
    */
    public boolean IsError;
    /**
    * 是否Http错误
    */
    public boolean IsHttpError;
    /**
    * Refresh Token
    */
    public String RefreshToken;
    /**
    * Token Type
    */
    public String TokenType;
}
